### DISCUSSION TITLE
安撫市場 英國新財相宣布撤回減稅方案

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#794] [General]
英國財政大臣因為減稅政策而辭職下台，新的財政大臣上任後馬上取消了減稅政策還可能調升稅率，在宣布後不但英磅上漲，英債的殖利率也跟著下跌，今年英國保守黨真的很多災多難，剛走了一個醜聞纏生的強森，又來一個讓英國債市崩跌的特拉斯，看來下次選舉保守黨很危險。
網址：https://www.google.com/amp/s/tw.news.yahoo.com/amphtml/%25E5%25AE%2589%25E6%2592%25AB%25E5%25B8%2582%25E5%25A0%25B4-%25E8%258B%25B1%25E5%259C%258B%25E6%2596%25B0%25E8%25B2%25A1%25E7%259B%25B8%25E5%25AE%25A3%25E5%25B8%2583%25E6%2592%25A4%25E5%259B%259E%25E6%25B8%259B%25E7%25A8%2585%25E6%2596%25B9%25E6%25A1%2588-140432595.html

**Comment by Pietra1226** [comment]
英國脫歐之後經濟狀況很難處哩，太多法規要重新權衡。希望他們趕快走出經濟困窘

**Comment by CPCpyt** [comment]
英國脫歐之後不管是人口或是經濟都有巨大的改變，
有太多的項目需要調整和規劃，希望英國能儘快離開頹勢。
